lib/mix/tasks/openapi.spec.yaml.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Mix.Tasks.Openapi.Spec.Yaml do
+  @moduledoc """
+  Serialize the given OpenApi spec module to a YAML file.
+
+  ## Examples
+
+      $ mix openapi.spec.yaml --spec PhoenixAppWeb.ApiSpec apispec.yaml
+
+  ## Command line options
+   * `openapi.spec.json` - includes available options
```

Comment:
This should reference a .yaml file here.

Reply:
Fixed